### PR TITLE
Initial project layout

### PR DESCRIPTION
**Pull request by mbaynton**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 fixtures/tufkeystore
 venv/
+
+/vendor/
```
Comment (heddn) [inline .gitignore:4]:
```suggestion
/vendor/
.idea
composer.lock
.DS_Store
.phpunit.result.cache
```

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "php-tuf/php-tuf",
+    "description": "PHP implementation of The Update Framework (TUF)",
+    "type": "library",
+    "require-dev": {
+        "phpunit/phpunit": "^4.8",
+        "squizlabs/php_codesniffer": "^3.5"
+    },
+    "license": "MIT",
+    "minimum-stability": "stable",
+    "require": {},
```
Comment (heddn) [inline composer.json:11]:
```suggestion
    "require": {
        "php": ">=5.3.0"
    },
```

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "php-tuf/php-tuf",
+    "description": "PHP implementation of The Update Framework (TUF)",
+    "type": "library",
+    "require-dev": {
+        "phpunit/phpunit": "^4.8",
```
Comment (heddn) [inline composer.json:6]:
```suggestion
        "phpunit/phpunit": "^4|^5|^6|^7|^8|^9",
        "symfony/phpunit-bridge": "^2|^3|^4|^5"
```

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "require": {},
+    "autoload": {
+        "psr-4": {
+            "Tuf\\": "src/"
+        }
+    },
+    "scripts": {
+        "code-sniff": [
+            "./vendor/bin/phpcs ./src"
+        ],
```
Comment (heddn) [inline composer.json:20]:
```suggestion
        "code-sniff": "phpcs ./src ./tests"
```

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        "test": [
+            "./vendor/bin/phpunit tests"
+        ],
```
Comment (heddn) [inline composer.json:23]:
```suggestion
        "test": "phpunit ./tests"
```

File: composer.lock [hunk 6]
```diff
@@ -0,0 +1,1253 @@
+{
+    "_readme": [
```
Comment (heddn) [inline composer.lock:2]: Remove lock file?

File: composer.json (hunk 3 above)
Comment (xjm) [inline composer.json:6]: We discussed that, although we want the client to support down to PHP 5.3 for various projects' legacy versions, we're probably better off writing the tests in a modern, secure, and forward-compatible version of PHPUnit to start. So I'd recommend PHPUnit 8 or 9 here as a dev dependency, and then backport the test suite to PHPUnit 4 as needed later. (We can still support PHP 5 for prod.)

Comment (heddn) [reply composer.json:6]: The issue is that we can't run CI then and certify that we support PHP 5.3+ if we don't also use older versions of PHP Unit

File: composer.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "php-tuf/php-tuf",
+    "description": "PHP implementation of The Update Framework (TUF)",
+    "type": "library",
+    "require-dev": {
+        "phpunit/phpunit": "^4|^5|^6|^7|^8|^9",
+        "symfony/phpunit-bridge": "^2|^3|^4|^5"
```
Comment (tedbow) [inline composer.json]: Needs comma

File: composer.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        "php": ">=5.3.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Tuf\\": "src/"
+        }
+    },
+    "scripts": {
+        "code-sniff": "phpcs ./src ./tests"
+        "test": "phpunit ./tests"
```
Comment (tedbow) [inline composer.json]: needs comma

**Comment by xjm** [comment]
Ted was able install it locally, so merging. Thanks!

**Pull request merged**